ffcx/element_interface.py

```diff
@@ -316,6 +335,11 @@ def cell_type(self):
         """Get the Basix cell type used to initialise the element."""
         return self._cell
 
+    @property
+    def discontinuous(self):
+        """Get a bool indicating whether the discontinuous version of the element is used."""
```

Comment:
Rather than putting `bool` in the docstring, could we make it a return type hint?

And should the function go into `BaseElement`?